File: 3783/Main.py
class Solution:
    def reverseNumber(self, n: int) -> int:
        revNum = 0
        while n > 0:
            revNum = revNum * 10 + (n % 10)
            n //= 10
        return revNum

    def mirrorDistance(self, n: int) -> int:
        return abs(self.reverseNumber(n) - n)

File: 3783/Main.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    int reverseNumber(int n) {
        int res = 0;
        while (n > 0) {
            res = res * 10 + n % 10;
            n /= 10;
        }
        return res;
    }

    int mirrorDistance(int n) { return abs(n - reverseNumber(n)); }
};